packages/datagateway-search/src/search/advancedHelpDialogue.component.test.tsx

```diff
@@ -39,12 +52,8 @@ describe('Advanced help dialogue component tests', () => {
   };
 
   it('renders correctly', () => {
-    useSelector.mockImplementation(() => {
-      return dgSearchInitialState;
-    });
-
-    const wrapper = shallow(<AdvancedHelpDialogue />);
-    expect(wrapper).toMatchSnapshot();
+    const { asFragment } = renderComponent({ initialState: state });
+    expect(asFragment()).toMatchSnapshot();
   });
 
   it('can open and close help dialogue', () => {
```

Comment:
It's a good idea to rewrite this test using RTL (although it may be being done through https://github.com/ral-facilities/datagateway/issues/1440 so if so, ignore this comment)

Reply:
Yep, in fact I have already migrated this test to RTL!